### ISSUE TITLE
Create multi-district idea

### ISSUE THREAD

**Issue by crisptrutski** [votepourca/geoforms#6]
From what I see so far, there's a control in each district view to display the "create idea form". How does this gel with ideas being potentially multidistrict?

Some options:
1. Deduplicate ideas between districts (extra work for users and codebase)
2. Click on the trigger control in multiple districts and then complete the form (confusing and awkward)
3. Add district toggle to the idea form

Since (3) seems sufficient in any case, I'm not sure I see the value in creating a from trigger inside each district.

**Comment by crisptrutski** [comment]
@votepourca it sounds like you're OK with ideas being limited to a single district for now?

**Comment by votepourca** [comment]
Yes. There are ideas like "Open a vegan restaurant" that could be found in more than one district. But supporters would be different. But you can also imagine a single idea that "crosses" more than one district. Like "Having an other cycle road that connects Manhattan to Brooklyn." For now, I have very few ideas I collected that implies this. So maybe a user cannot do it, but an admin yes? Could the data still accepts a collection of districts?
